Vagrantfile

```diff
@@ -195,8 +195,13 @@ Vagrant.configure("2") do |config|
       inline: BOOTSTRAP
   end
 
-  (1..9).each do |i|
-    node_name = "node#{i}"
+  INSTALL_PYTHON = <<-SCRIPT
+  #!/bin/bash
+  apt install python -y
+  SCRIPT
+
+  (1..5).each do |i|
+    node_name = "centos#{i}"
```

Comment:
If I look at the entire file, you're missing the box/version definitions here.

```
  node.vm.box = "centos/7"
  node.vm.box_version = "1811.02"
```

Reply:
Is it a requirement ? I supposed it was defined by default  at the line 158 in the vagrant.configure

Reply:
Oh, indeed. It wouldn't hurt to repeat it here (or use the loop version) instead of relying on default propagation.

Reply:
I use loop, thank you for your help.